### PR TITLE
chore: standardise doc-strings for newly added API

### PR DESCRIPTION
**Pull request by wwestgarth**
Only changes are doc-strings in the protos.

Mostly just fixing up things like missing full-stops, and using standard phrases used across all existing APIs

### CODE REVIEW

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 1]
```diff
@@ -475,23 +475,23 @@ service TradingDataService {
 
   // List liquidity provisions
   //
-  // Get a list of liquidity provisions for a given market
+  // Get a list of liquidity provisions for a given market.
   rpc ListLiquidityProvisions(ListLiquidityProvisionsRequest) returns (ListLiquidityProvisionsResponse) {
-    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Liquidity provisions"};
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:480]: There's also a "liquidity providers" tag that has 1 endpoint in it. I wonder if these two should be combined. https://docs.vega.xyz/testnet/category/api/rest/data-v2/liquidity-providers

Comment (wwestgarth) [reply protos/sources/data-node/api/v2/trading_data.proto:480]: Good spot, I'll add that in too.

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 2]
```diff
@@ -3708,16 +3711,15 @@ message ReferralProgram {
...
-  // Number of epochs over which to evaluate a referral set's running volume.
+  // Number of epochs over which the referral set's running volume is evaluated.
   uint64 window_length = 5;
-  // Defined staking tiers in increasing order. First element will give Tier 1,
-  // second element will give Tier 2, and so on. Determines the level of
+  // Defined benefit tiers ordered by increasing reward multiplier. Determines the level of
   // benefit a party can expect based on their staking.
   repeated vega.StakingTier staking_tiers = 6;
-  // Timestamp as Unix time in nanoseconds at which the program ended.
+  // Timestamp in Unix nanoseconds, at which the program ended.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3722]:
```suggestion
  // Timestamp, in Unix nanoseconds, when the program ended.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 3]
```diff
@@ -3727,9 +3729,9 @@ message ReferralSet {
   string id = 1;
   // Party that created the set.
   string referrer = 2;
-  // Time in Unix nanoseconds when the set was created.
+  // Timestamp in Unix nanoseconds, when the set was created.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3732]:
```suggestion
  // Timestamp, in Unix nanoseconds, when the set was created.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 4]
```diff
@@ -3727,9 +3729,9 @@ message ReferralSet {
...
   int64 created_at = 3;
-  // Time in Unix nanoseconds when the set was updated.
+  // Timestamp in Unix nanoseconds, when the set was updated.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3734]:
```suggestion
  // Timestamp, in Unix nanoseconds, when the set was updated.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 5]
```diff
@@ -3773,13 +3775,13 @@ message ReferralSetReferee {
   string referral_set_id = 1;
   // Party that joined the set.
   string referee = 2;
-  // Time in Unix nanoseconds when the party joined the set.
+  // Timestamp in Unix nanoseconds when the party joined the set.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3778]:
```suggestion
  // Timestamp, in Unix nanoseconds, when the party joined the set.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 6]
```diff
@@ -3822,15 +3823,13 @@ message ListReferralSetRefereesResponse {
 
 // Request to get a referral set's stats
 message GetReferralSetStatsRequest {
-  // Unique ID of the set.
+  // Restrict referral set statisitcs to those for the given referral set.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3826]:
```suggestion
  // Restrict referral set statistics to those for the given referral set.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 7]
```diff
@@ -3869,11 +3868,11 @@ message ReferralSetStats {
...
   // Reward factor applied to the party.
   string reward_factor = 5;
-  // Current referee notional taker volume
+  // Current referee notional taker volume.
   string epoch_notional_taker_volume = 6;
-  // The multiplier applied to the referral reward factor when calculating referral rewards due to the referrer
+  // Multiplier applied to the referral reward factor when calculating referral rewards due to the referrer.
   string rewards_multiplier = 7;
-  // The proportion of the referees taker fees to be rewarded to the referrer
+  // Proportion of the referees taker fees to be rewarded to the referrer.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3875]:
```suggestion
  // Proportion of the referee's taker fees to be rewarded to the referrer.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 8]
```diff
@@ -3889,17 +3888,17 @@ message Team {
   optional string team_url = 4;
   // Link to an image of the team's avatar.
   optional string avatar_url = 5;
-  // Time in Unix nanoseconds when the team was created.
+  // Timestamp in Unix nanoseconds when the team was created.
   int64 created_at = 6;
-  // Tells if a party can join the team or not.
+  // Whether the team is closed to new party members.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3893]:
```suggestion
  // Whether or not the team is closed to new party members.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 9]
```diff
@@ -3915,11 +3914,11 @@ message TeamConnection {
 
 // Request to list all teams
 message ListTeamsRequest {
-  // Optional team ID to filter for a specific team
+  // Restrict teams to those with the given team ID.
   optional string team_id = 1;
-  // Optional party ID to filter by a specific referrer, or referee
+  // Restrict teams to those where the given party is a referrer ot a referee.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3919]:
```suggestion
  // Restrict teams to those where the given party is a referrer or a referee.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 10]
```diff
@@ -3915,11 +3914,11 @@ message TeamConnection {
...
   optional string party_id = 2;
-  // Optional pagination information to limit the data that is returned.
+  // Pagniation controls.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3921]:
```suggestion
  // Pagination controls.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 11]
```diff
@@ -3933,7 +3932,7 @@ message ListTeamsResponse {
 message ListTeamRefereesRequest {
   // Team ID to list referees for.
   string team_id = 1 [(google.api.field_behavior) = REQUIRED];
-  // Optional pagination control
+  // Pagniation controls.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:3935]:
```suggestion
  // Pagination controls.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 12]
```diff
@@ -4013,19 +4012,19 @@ message ListTeamRefereeHistoryResponse {
 
 // Request that is sent to retrieve fees statistics.
 message GetFeesStatsRequest {
-  // Market ID to get referral fee statistics for. If omitted, asset ID must be provided.
+  // Restrict fee statistics to those related for the given market.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:4015]:
```suggestion
  // Restrict fee statistics to those related to the given market.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 13]
```diff
@@ -4013,19 +4012,19 @@ message ListTeamRefereeHistoryResponse {
...
   optional string market_id = 1;
-  // Asset ID to get referral fee statistics for. If omitted, market ID must be provided.
+  // Restrict fee statistics to those related for the given asset.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:4017]:
```suggestion
  // Restrict fee statistics to those related to the given asset.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 14]
```diff
@@ -4040,18 +4039,17 @@ message GetCurrentVolumeDiscountProgramResponse {
...
 message GetVolumeDiscountStatsRequest {
-  // Epoch at which the statistics apply. If not set, last
-  // epoch is used.
+  // Epoch to get volume discount statistics for. If not set, last epoch is used.
   optional uint64 at_epoch = 1;
-  // Party ID to return volume discount statistics for.
+  // Restrict volume discount statistics to those for the given party.
   optional string party_id = 2;
-  // pagination information to limit the data that is returned.
+  // Pagination information to limit the data that is returned.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:4046]: Should this one just get trimmed down to "Pagination controls" too?

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 15]
```diff
@@ -4075,32 +4073,29 @@ message VolumeDiscountStatsEdge {
...
+  // Defined benefit tiers ordered by increasing discounts. Determines the level of
   // benefit a party can expect based on performance criteria.
   repeated vega.VolumeBenefitTier benefit_tiers = 3;
-  // Timestamp as Unix time in nanoseconds, after which when the current epoch
-  // ends, the programs will end and benefits will be disabled.
+  // Timestamp in Unix nanoseconds, after which when the current epoch
+  // ends, the program will end and benefits will be disabled.
   int64 end_of_program_timestamp = 4;
-  // Number of epochs over which to evaluate a volume discount statistics' running volume.
+  // Number of epochs over which a volume discount statistics' running volume is evaluated.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:4097]:
```suggestion
  // Number of epochs over which the statistics for a volume discount program's running volume is evaluated.
```
Please double-check this rewording

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 16]
```diff
@@ -4075,32 +4073,29 @@ message VolumeDiscountStatsEdge {
...
   uint64 window_length = 5;
-  // Timestamp as Unix time in nanoseconds at which the program ended.
+  // Timestamp in Unix nanoseconds, at which the program ended.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:4099]:
```suggestion
  // Timestamp, in Unix nanoseconds, when the program ended.
```

File: protos/sources/vega/vega.proto [hunk 17]
```diff
@@ -1748,29 +1745,27 @@ message VolumeDiscountProgram {
...
   // Unique ID generated from the proposal that created this program.
   string id = 2;
-  // Defined benefit tiers in increasing order. First element will give Tier 1,
-  // second element will give Tier 2, and so on. Determines the level of
-  // benefit a party can expect based on performance criteria.
+  // Defined benefit tiers ordered by increasing discounts.
   repeated VolumeBenefitTier benefit_tiers = 3;
-  // Timestamp as Unix time in seconds, after which when the current epoch
-  // ends, the programs will end and benefits will be disabled.
+  // Timestamp in Unix nanoseconds, after which when the current epoch
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1750]: This is definitely nanoseconds, right?

File: protos/sources/vega/vega.proto [hunk 18]
```diff
@@ -1748,29 +1745,27 @@ message VolumeDiscountProgram {
...
+  // ends, the program will end and benefits will be disabled.
   int64 end_of_program_timestamp = 4;
-  // Number of epochs over which to evaluate a referral set's running volume.
+  // Number of epochs over a referral set's running volume is evaluated.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1753]:
```suggestion
  // Number of epochs over which a referral set's running volume is evaluated.
```

File: protos/sources/vega/vega.proto [hunk 19]
```diff
@@ -1748,29 +1745,27 @@ message VolumeDiscountProgram {
...
   repeated ActivityStreakBenefitTier tiers = 1;
 }
 
 // An activity streak benefit tier
 message ActivityStreakBenefitTier {
-  // The minimum number of epochs necessary for this tier.
+  // Number of epochs a party must be active to receive the multiplier.
   uint64 minimum_activity_streak = 1;
-  // The reward multiplier applicable for this tier.
+  // Reward multiplier applicable for this tier.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1767]:
```suggestion
  // Reward multiplier applicable to this tier.
```

File: protos/sources/vega/vega.proto [hunk 20]
```diff
@@ -1748,29 +1745,27 @@ message VolumeDiscountProgram {
...
   string reward_multiplier = 2;
-  // The vesting bonus applicable for this tier.
+  // Vesting bonus applicable for this tier.
```
Comment (candida-d) [inline protos/sources/vega/vega.proto:1769]:
```suggestion
  // Vesting bonus applicable to this tier.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 21]
```diff
@@ -808,24 +810,27 @@ service TradingDataService {
 
   // Get fees statistics
   //
-  // Gets accumulated fees, rewards, and applied discount information for a given asset or market for the latest epoch
-  // or a specific epoch.
+  // Gets accumulated fees, rewards, and applied discount information. Either a market or an asset must be supplied as filter.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:813]:
```suggestion
  // Get accumulated fees, rewards, and applied discount information. Either a market or an asset must be supplied as filter.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 22]
```diff
@@ -808,24 +810,27 @@ service TradingDataService {
...
+  // Get the current volume discount program for the network. This program may not be active if it has ended
+  // and has not been replaced by another.
   rpc GetCurrentVolumeDiscountProgram(GetCurrentVolumeDiscountProgramRequest) returns (GetCurrentVolumeDiscountProgramResponse) {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Volume discount program"};
   }
 
-  // Get the volume discount statistics for a given epoch for all parties
+  // Get volume discount statistics
+  //
+  // Get the information about a party's running traded volume, the discount factor it earns them.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:828]:
```suggestion
  // Get the information about a party's running traded volume, and the discount factor it earns them.
```

File: protos/sources/vega/vega.proto (hunk 17 above)
Comment (wwestgarth) [reply protos/sources/vega/vega.proto:1750]: Whoops, this one is seconds still, good spot.

**Review by jeremyletang — DISMISSED** [review]

**Review by AnExsomnis — APPROVED** [review]

**Pull request merged**